### ISSUE TITLE
snakebacon v0.0.1 compiles but doesn't properly install on MacOSX

### ISSUE THREAD

**Issue by brews** [brews/snakebacon#1]
snakebacon v0.0.1 compiles but doesn't properly install on MacOSX.

First, I downloaded source from the release. Created `test_env` in Conda and activate it. Run:

`python setup.py build_ext --inplace
`
to compile cython and C/C++.

I then ran unittests on the locally-compiled code and everything seemed fine.

Local package install with:

`python setup.py install --user
`
switched to a clean directory and tested the installed package in ipython. Got the error on import.

Here is the error in an ipython session:

```

> Python 3.5.3 |Continuum Analytics, Inc.| (default, Mar  6 2017, 12:15:08) 
> Type 'copyright', 'credits' or 'license' for more information
> IPython 6.1.0 -- An enhanced Interactive Python. Type '?' for help.
> 
> In [1]: import snakebacon as sb
> ---------------------------------------------------------------------------
> ImportError                               Traceback (most recent call last)
> <ipython-input-1-4dc5dfae81e0> in <module>()
> ----> 1 import snakebacon as sb
> 
> ~/.local/lib/python3.5/site-packages/snakebacon-0.0.1-py3.5-macosx-10.7-x86_64.egg/snakebacon/__init__.py in <module>()
> ----> 1 from .agedepth import AgeDepthModel
>       2 from .records import CalibCurve, DateRecord, ProxyRecord, DatedProxyRecord
>       3 from .records import read_14c, read_dates, read_proxy
>       4 
>       5 try:
> 
> ~/.local/lib/python3.5/site-packages/snakebacon-0.0.1-py3.5-macosx-10.7-x86_64.egg/snakebacon/agedepth.py in <module>()
>       4 import numpy as np
>       5 
> ----> 6 from .mcmc import McmcSetup
>       7 from .records import DatedProxyRecord
>       8 
> 
> ~/.local/lib/python3.5/site-packages/snakebacon-0.0.1-py3.5-macosx-10.7-x86_64.egg/snakebacon/mcmc.py in <module>()
>       4 
>       5 from snakebacon.records import DateRecord
> ----> 6 from snakebacon.bacon import run_baconmcmc
>       7 
>       8 
> 
> ~/.local/lib/python3.5/site-packages/snakebacon-0.0.1-py3.5-macosx-10.7-x86_64.egg/snakebacon/bacon/__init__.py in <module>()
>       4 """
>       5 
> ----> 6 from snakebacon.bacon.baconwrap import run_baconmcmc
> 
> ImportError: dlopen(/Users/sbm/.local/lib/python3.5/site-packages/snakebacon-0.0.1-py3.5-macosx-10.7-x86_64.egg/snakebacon/bacon/baconwrap.cpython-35m-darwin.so, 2): Library not loaded: @rpath/libgsl.19.dylib
>   Referenced from: /Users/sbm/.local/lib/python3.5/site-packages/snakebacon-0.0.1-py3.5-macosx-10.7-x86_64.egg/snakebacon/bacon/baconwrap.cpython-35m-darwin.so
>   Reason: image not found
```

**Comment by brews** [comment]
The key here seems to be `Library not loaded: @rpath/libgsl.19.dylib`. Looks like it is having trouble with GSL when installed from Conda.

**Comment by brews** [comment]
Install seems to work if we use the `conda-forge` channel when creating our virtual environment.

I think we can add this to our testing environments. Sounds like it might be trickier when building the actual package. See: https://github.com/conda/conda/issues/988 and https://github.com/conda/conda-build/issues/532.

**Comment by brews** [comment]
This is closed with 65e0143.
